projects/TG-DMR190101.yaml

```
@@ -0,0 +1,12 @@
+Name: TG-DMR190101
+Description: 1D Nanoconfined Helium: A Versatile Platform for Exploring
```

```suggestion
Description: "1D Nanoconfined Helium: A Versatile Platform for Exploring"
```
There's a colon in there so you need to quote it.